ooi/wsgi/__init__.py

```diff
@@ -61,15 +61,9 @@ def get_best_match_content_type(self, default_match=None):
         return content_type
 
     def get_parser(self):
-        mtype = parsers.get_media_map().get(self.get_content_type,
-                                            "header")
+        mtype = parsers.get_media_map().get(self.get_content_type, "header")
```

Comment:
See 4147ccc66ca2b249e2dcc97aaefa967b69d3b413, should be `self.get_content_type()`